integration-tests/features/steps/three_scale_preview.py

```diff
@@ -74,8 +117,44 @@ def npm_manifest_stack_analysis(context, manifest, version=3, user_key="without"
     """Send the NPM package manifest file to the stack analyses."""
     endpoint = threescale_preview_endpoint_url(context, 'stack-analyses')
     use_user_key = parse_token_clause(user_key)
-    send_manifest_to_stack_analyses(context, manifest, 'package.json',
-                                    endpoint, use_user_key, rate)
+    send_manifest_to_stack_analyses(context, manifest, 'npmlist.json',
+                                    endpoint, use_user_key, rate, ecosystem='npm', origin='vscode')
+
+
+@when("I send Python package manifest {manifest} to stack analysis through "
+      "3scale gateway {user_key} user_key")
+@when("I send Python package manifest {manifest} to stack analysis {rate:d} times"
+      " in a minute through 3scale gateway {user_key} user_key")
+def python_manifest_stack_analysis(context, manifest, version=3, user_key="without", rate=1):
+    """Send the NPM package manifest file to the stack analyses."""
+    endpoint = threescale_preview_endpoint_url(context, 'stack-analyses')
+    use_user_key = parse_token_clause(user_key)
+    send_manifest_to_stack_analyses(context, manifest, 'pylist.json',
+                                    endpoint, use_user_key, rate, ecosystem='pypi', origin='vscode')
+
+
+@when("I test {ecosystem} dependencies file {manifest} for stack analysis from {origin} "
+      "through 3scale gateway {user_key} user_key")
+def scale_process_deps_file(context, ecosystem, manifest, origin, rate=1,
+                            user_key="without"):
+    """Test stack analyses of an ecosystem specific dependencies file from an integration point."""
+    endpoint = threescale_preview_endpoint_url(context, 'stack-analyses')
+    use_user_key = parse_token_clause(user_key)
+    test_stack_analyses_with_deps_file(context, manifest, endpoint, use_user_key,
+                                       rate, ecosystem.lower(), origin)
+
+
+@when("I send Maven package manifest {manifest} to stack analysis through "
+      "3scale gateway {user_key} user_key")
+@when("I send Maven package manifest {manifest} to stack analysis {rate:d} times"
+      " in a minute through 3scale gateway {user_key} user_key")
+def maven_manifest_stack_analysis(context, manifest, version=3, user_key="without", rate=1):
```

Comment:
Can we skip doing load testing in these test cases

Reply:
the same function is also used during load test so if we specify a rate it will work otherwise it will just make one request